modules/lib/src/main/resources/assets/js/app/settings/dialog/ProjectSelectionDialog.ts

```diff
@@ -137,4 +139,19 @@ export class ProjectSelectionDialog
         });
     }
 
+    // Cycle to the first project element when tabbing in the last element
+    private setLastProjectCycleOnTabEvent(): void {
+        if(this.projectsList.getItemViews().length === 0) return;
+
+        const firstProject: ProjectListItem =
+            this.projectsList.getItemViews()[0] as ProjectListItem;
```

Comment:
Everything is fine in here, just let's use our commonly used angle-bracket syntax:

`const firstProject: ProjectListItem = <ProjectListItem>this.projectsList.getItemViews()[0];`